docs/install/install.rst

```diff
@@ -1,51 +1,71 @@
 .. meta::
-  :description: TransferBench documentation 
+  :description: TransferBench documentation
   :keywords: TransferBench, API, ROCm, HIP
 
+.. _install-transferbench:
+
 ---------------------------
-TransferBench installation
+Installation
 ---------------------------
 
-The following software is required to install TransferBench:
+This document provides information required to install and build TransferBench.
 
-* ROCm stack installed on the system (HIP runtime)
-* `libnuma` installed on the system
+Prerequisite
+---------------
 
---------------------------
-Building TransferBench
---------------------------
+* Install ROCm stack on the system (HIP runtime)
+* Install ``libnuma`` on the system
+* Enable AMD IOMMU and set to passthrough for AMD Instinct cards
```

Comment:
```suggestion
* Enable AMD IOMMU and set to passthrough for AMD Instinct accelerators
```

Reply:
Should we provide an xref for more info here?

Reply:
@gilbertlee-amd which one of these links can we use here? 
https://rocm.docs.amd.com/en/develop/how-to/system-optimization/mi300x.html?highlight=IOMMU#iommu-configuration-systems-with-256-cpu-threads

https://rocm.docs.amd.com/en/develop/how-to/system-optimization/mi200.html?highlight=IOMMU#systems-with-256-cpu-threads-iommu-configuration

https://rocm.docs.amd.com/en/develop/how-to/system-optimization/mi100.html#systems-with-256-cpu-threads-iommu-configuration